RadarSDK/remodel/RadarLocation.value

```diff
@@ -0,0 +1,5 @@
+RadarLocation {
+    double latitude
+    double longtitude
+    RadarLocationType(NSInteger) type
```

Comment:
Add a RadarLocation model to demo the support of enum and the nested model class